data/transport missions.txt

```diff
@@ -1868,6 +1872,310 @@ mission "Terraforming 5"
+
+
+
+event "terraforming Tundra"
+	planet "Tundra"
+		description `Millions of years ago, this was a warm world, perhaps even tropical. But some cataclysmic event, perhaps a meteor strike or a massive volcanic eruption, altered the planet's atmosphere enough to turn it into the nearly lifeless, frozen planet it is today. Recent experiments in terraforming the planet have attempted to remedy the climate by causing a supervolcano to prematurely erupt.`
+		description `	The first settlers came here to drill for the oil trapped deep under the surface, a relic of Tundra's former, more lively days. Instead of refining the oil into plastic, which sells relatively cheap in this region, they have developed an industry in synthetic fabrics and clothing.`
+
+
+
+event "terraforming timer 2"
+
+
+
+mission "Terraforming Follow-up"
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
+		near "Sol" 100
+		attributes spaceport
+	to offer
+		has "event: terraforming timer 2"
+	on offer
+		event "terraforming follow-up"
+		conversation
+			`When you land, you receive a message from Amy, the girl who you helped to terraform Rand and Tundra.`
+			`	"I hope everything is going well, Captain. I just wanted to give you an update on what has occurred in the past year.`
+			`	"Tundra's climate has drastically improved over the past year. Areas around the equator are starting to become completely snow free, and I was amazed when I was able to walk outside without a heavy coat on. The Republic has placed a halt on any more experiments, as they wish to see how the situations on Tundra and Rand turn out.`
+			`	"Speaking of Rand, last month I got to catch up with Eric and Alaric when I visited the planet. They've told me that Rand is becoming a far better place to live on now. It's even becoming a popular tourist destination. They wanted me to relay their gratitude to you for making their dream come true.`
+			`	"That's all I have to say. Thanks again for all the help you've provided to me. I don't know where I would be without it."`
+				decline
+
+
+
+event "terraforming follow-up"
+	planet "Rand"
+		add attribute "tourism"
+		description `Rand is a desert world, too dry for much farming and with gravity low enough to be uncomfortable for most human beings. It is, however, the best source of heavy metals in the galactic south. Aside from the managers of the mining companies, nearly all the people here are migrant workers from elsewhere in the Dirt Belt, who have come to spend a season working for the relatively high wages that uranium mining offers, either to send money off-world or to save it up in order to build a better life for themselves.`
+		description `	Due to the results of recent terraforming experiments, Rand has become a far more habitable place. Fascination of this rapid change in climate has lead to Rand becoming a popular tourist destination of the Dirt Belt.`
```

Comment:
I really don't think it makes sense for Rand's climate to change this quickly. And in any case, there's a typo in the last sentence (maybe you meant "Fascination with..."?).